.github/pull_request_template.md

```diff
@@ -1,12 +1,38 @@
----
-name: Pull request template
-about: 'type(scope): Subject of the pull request '
----
+<!--
```

Comment:
Are you sure you want to include this in the template and not in the contributing/developer guidelines?

Reply:
Yes. We should also update the developer guidelines but regardless, this is the place we know with certitude our users will read, more detailed information could be added to other docs and we should link them to those docs but being honest, how many times have you (personally) follow those links?